tests/system/Helpers/URLHelperTest.php

```diff
@@ -1250,4 +1250,64 @@ public function testBasedWithOtherIndex()
 		$this->assertEquals('http://example.com/ci/v4/controller/method', base_url('controller/method', null, $config));
 	}
 
+	/**
+	 * @dataProvider urlToProvider
+	 */
+	public function testUrlTo(string $expected, string $input, ...$args)
+	{
+		$_SERVER['HTTP_HOST'] = 'example.com';
+
+		$routes = service('routes');
+		$routes->add('path/(:any)/to/(:num)', 'myController::goto/$1/$2', ['as' => 'gotoPage']);
+		$routes->add('route/(:any)/to/(:num)', 'myOtherController::goto/$1/$2');
+
+		$this->assertEquals($expected, url_to($input, ...$args));
+	}
+
+	/**
+	 * @dataProvider urlToMissingRoutesProvider
+	 */
+	public function testUrlToThrowsOnEmptyOrMissingRoute(string $route)
+	{
+		$this->expectException(\CodeIgniter\Router\Exceptions\RouterException::class);
+
+		url_to($route);
+	}
+
+	public function urlToProvider()
+	{
+		if (config('App')->indexPage != '') {
```

Comment:
The opening braces should be on its own line.

Reply:
Fixed it